File: ContextoDB/Models/Catalogos/RolUsuario.cs

namespace ContextoDB.Models.Catalogos
{
    public  class RolUsuario
    {
        public int RolId { get; set; }
        public int UsuarioId { get; set; }
        public bool Activo { get; set; }
        public Rol Rol { get; set; }
        public Usuario Usuario { get; set; }
    }
}

File: CursoTemplate/ClientApp/src/app/data/model/catalogo/rol-usuario.ts
import { ThemePalette } from "@angular/material/core";

export interface RolUsuario {
  rolNombre: string;
  rolId: number;
  nombreUsuario: string;
  nombreCompleto: string;
  usuarioId: number;
  activo: boolean;
}


export interface RolUsuarioListCheck {
  name: string;
  completed: boolean;
  color: ThemePalette;
  listaRol?: RolUsuario[];
}
